### PR TITLE
Issue #169 Table editor mark search

### PR DESCRIPTION
**Pull request by chelovek-18**

### CODE REVIEW

**Comment by duker33** [comment]
@chelovek-18 , this PR is not just fix. It closes task stb#169, as i suppose.
Highlight this task plz.

And your PR should resolve task. It means, that you should integrate this new version with stb code.
How to do it:
- create new refarm-site's release of release page: https://github.com/fidals/refarm-site/releases
- fix stb deps in separated PR: https://github.com/fidals/stroyprombeton/blob/master/requirements.txt#L19

**Comment by chelovek-18** [comment]
@duker33, Thanks :)

File: generic_admin/front/js/components/tableEditor/jqGrid.es6 [hunk 1]
```diff
@@ -6,6 +6,7 @@ class TableEditor {
 
     this.filterFields = [
       'name',
+      'mark',
```
Comment (duker33) [inline generic_admin/front/js/components/tableEditor/jqGrid.es6]: not every site has `mark` field in it's models. Rm it plz

**Comment by duker33** [comment]
@chelovek-18 , we release only commits merged on master, so removed `0.1.4`.
stb#169 doesn't require this release, as we understood. So, doesn't matter.

Release process is complicated and that's why we don't describe it. Just be lazy :)

File: generic_admin/front/js/components/tableEditor/jqGrid.es6 (hunk 1 above)
Comment (chelovek-18) [reply generic_admin/front/js/components/tableEditor/jqGrid.es6]: в jqGrid.es6 (строка 214) ищем по заданному массиву filterField на фронтенде - если такого поля нет в таблице, то поиска по нему просто не будет.

А включить в таблицу его можно в default_filters для класса TableEditor - либо вообще для refarm_site в refarm_site/generic_admin/views.py (что я не стал делать), либо в stroyprombeton/stroyprombeton/views/admin.py

Comment (duker33) [reply generic_admin/front/js/components/tableEditor/jqGrid.es6]: you should inherit the custom class from TableEditor on STB side.
And redefine its fields.
See `class SETableEditor` at `front/js/components/admin.es6`  at SE repo for example

**Comment by chelovek-18** [comment]
though correction in the file colModel.es6 has no relation to a task...

**Pull request merged**